src/converters/odom.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2015 Aldebaran
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+/*
+* LOCAL includes
+*/
+#include "odom.hpp"
+#include "../tools/from_any_value.hpp"
+
+/*
+* BOOST includes
+*/
+#include <boost/foreach.hpp>
+#define for_each BOOST_FOREACH
+
+
+#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+
+namespace naoqi
+{
+namespace converter
+{
+ 
+OdomConverter::OdomConverter( const std::string& name, const float& frequency, const qi::SessionPtr& session ):
+  BaseConverter( name, frequency, session ),
+  p_motion_( session->service("ALMotion") )
+  
+{
+}
+
+void OdomConverter::registerCallback( message_actions::MessageAction action, Callback_t cb )
+{
+  callbacks_[action] = cb;
+}
+
+void OdomConverter::callAll( const std::vector<message_actions::MessageAction>& actions )
+{
+  
+  int FRAME_WORLD = 1;
```

Comment:
please, cite the reference at http://doc.aldebaran.com/2-1/naoqi/motion/control-cartesian.html in some comments.